### DISCUSSION TITLE
Contribute

### DISCUSSION THREAD

**Discussion by shajin-sha** [kiqpo/kiqpo#28] [General]
### Hey @TarunavBA how it's going ?
Hope you're doing well.

#### You may start to contribute **kiqpo**.
Let's start it from today itself, are you ready.?

We have many **TODO** in kiqpo to work around, do something

- Network (Get/Post) request.
- Add styleing to NavBar.
- Add manifest.json (pwa).
- Add other HTML elements (span,li, ect..)

You may ask any questions to me here

Let's start the day.😉😊

**Comment by shajin-sha** [comment]
You may also add Todo.md file which contains all Todo list (that comes in to your thought,as i mentioned there)

**Comment by TarunavBA** [comment]
Hey, @shajin-sha looks like Kiqpo has grown so much!

**Reply by shajin-sha** [reply]
Are you kidding?
